File: backend/ClientService/src/main/java/com/example/demo/entity/RequirementSkillTable.java
package com.example.demo.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "requirementskill_table")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class RequirementSkillTable {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "reqskill_id")
    private Integer reqSkillId;

    @ManyToOne
    @JoinColumn(name = "req_id", nullable = false)
    private RequirementTable requirement;

    @ManyToOne
    @JoinColumn(name = "skill_id", nullable = false)
    private SkillTable skill;
}

File: backend/admin_panel/Models/RequirementskillTable.cs
using System;
using System.Collections.Generic;

namespace admin_panel.Models;

public partial class RequirementskillTable
{
    public int ReqskillId { get; set; }

    public int ReqId { get; set; }

    public int SkillId { get; set; }

    public virtual RequirementTable Req { get; set; } = null!;

    public virtual SkillTable Skill { get; set; } = null!;
}
